config/src/main/kotlin/maru.config/Config.kt

```diff
@@ -37,29 +38,41 @@ data class P2P(
 )
 
 data class Validator(
-  val key: ByteArray,
-  val client: ValidatorClientConfig,
+  val privateKey: ByteArray,
+  val engineApiClient: ApiEndpointConfig,
```

Comment:
Don't the engine API should be part of the validator config. IMHO it's not specific to the validator at all. Fine with doing any config changes in another PR though.

Reply:
A complete configuration layout proposal would be nice, because if Engine API isn't a part of the validator, then application structure should change as well